sql/core/src/test/scala/org/apache/spark/sql/streaming/ReportSinkMetricsSuite.scala

```diff
@@ -94,11 +99,10 @@ class ReportSinkMetricsSuite extends StreamTest {
       TestSinkTable
     }
 
-    def createRelation(
-                        sqlContext: SQLContext,
-                        mode: SaveMode,
-                        parameters: Map[String, String],
-                        data: DataFrame): BaseRelation = {
+    def createRelation(sqlContext: SQLContext,
```

Comment:
probably the last nit: the indentation rule Spark uses is quite different with others, hence you'd like to refer to the Scala style guide.

https://github.com/databricks/scala-style-guide

https://github.com/databricks/scala-style-guide#spacing-and-indentation

> For method declarations, use 4 space indentation for their parameters and put each in each line when the parameters don't fit in two lines. Return types can be either on the same line as the last parameter, or start a new line with 2 space indent.

Below is the correct indentation for this case.

```
def createRelation(
    sqlContext: SQLContext,
    mode: SaveMode,
    parameters: Map[String, String],
    data: DataFrame): BaseRelation = {
```

Reply:
fix